.env.test

```diff
@@ -1,23 +1,16 @@
-# direnv will make a copy of this file as an .env file
-# Follow the instructions in README.md as how to add your personal
-# tokens and secrets in this file
```

Comment:
We don't need all this boilerplate in the test env, it's copy-pasta from the example env.